File: 0448 - Find All Numbers Disappeared in an Array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        for i in range(len(nums)):
            integer = abs(nums[i]) - 1
            nums[integer] = -1 * abs(nums[integer])
        result = []
        for i in range(len(nums)):
            if nums[i] > 0:
                result.append(i + 1)
        return result

File: 0448 - Find All Numbers Disappeared in an Array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int> result;
        for(int i = 0; i < nums.size(); i++){
            int integer = abs(nums[i]) - 1;
            nums[integer] = abs(nums[integer]) * -1;
        }
        for(int i = 0; i < nums.size(); i++){
            if(nums[i] > 0){
                result.push_back(i + 1);   
            }
        }
        return result;
    }
};